src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,10 @@
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User user = new UserService().findByEmail(email);
+        if (user != null && user.getPassword().equals(password)) {
+            return true;
+        }
         return false;
```

Comment:
Can be simplified to:
```suggestion
        return user != null && user.getPassword().equals(password);
```